lib/improver/generate_ancillaries/generate_ancillary.py

```diff
@@ -81,33 +81,36 @@ def _make_mask_cube(mask_data, key, coords,
     return mask_cube
 
 
-def find_standard_ancil(grid, stage_ancil):
+def find_standard_ancil(standard_ancil, msg=None):
     """
-    Reads standard ancillary from StaGE or raises exception.
+    Reads standard ancillary or raises exception.
 
     Parameters
     -----------
-    grid : string
-      String describing standard grid
+    standard_ancil : string
```

Comment:
I think this variable name could be changed, as it's a bit confusing that `standard_ancil` is both a path, and later represents a cube.